oauthproxy.go

```diff
@@ -761,48 +746,48 @@ func (p *OAuthProxy) OAuthCallback(rw http.ResponseWriter, req *http.Request) {
 	err := req.ParseForm()
 	if err != nil {
 		logger.Errorf("Error while parsing OAuth2 callback: %v", err)
-		p.ErrorPage(rw, req, http.StatusInternalServerError, "Internal Server Error", err.Error())
+		p.ErrorPage(rw, req, http.StatusInternalServerError, err.Error())
 		return
 	}
 	errorString := req.Form.Get("error")
 	if errorString != "" {
 		logger.Errorf("Error while parsing OAuth2 callback: %s", errorString)
-		p.ErrorPage(rw, req, http.StatusForbidden, "Permission Denied", errorString)
+		p.ErrorPage(rw, req, http.StatusForbidden, errorString, "Login Failed: The upstream identity provider returned an error.")
```

Comment:
I think this case is safe to not obfuscate and show the full error in the UI (debug or not) -- it is already part of the querystring in the request URL the enduser has if they got an error back from the IdP `authorization_endpoint`

Reply:
ToDo: Create message var and duplicate the error string in both debug and non debug cases